sklearn/tests/test_multiclass.py

```diff
@@ -540,3 +544,34 @@ def test_ecoc_gridsearch():
     cv.fit(iris.data, iris.target)
     best_C = cv.best_estimator_.estimators_[0].C
     assert_true(best_C in Cs)
+
+
+def test_ovr_svc_issue_5495():
```

Comment:
I wouldn't name it after the issue and I think the test should go into the SVC tests.
Also, your test is much too complicated.